tool/docs/adoc/AsciiDocTableBuilder.kt

```diff
@@ -32,7 +32,7 @@ class AsciiDocTableBuilder(private val headers: List<String>) {
     }
 
     private fun header(): String {
-        return "[cols=\"~" + ",~".repeat(this.headers.size - 1) +
+        return "[cols=\"" + ",".repeat(this.headers.size - 1) +
```

Comment:
Vladimir asked me to remove these symbols as we can't understand what they do and they just distract us while debugging tables' behavior.

Reply:
ok